src/components/language-switch/language-switch.tsx

```diff
@@ -18,26 +23,30 @@ export const LanguageSwitch = ({
   translation,
   className = 'language-switch',
 }) => {
-  const { languages, language, originalPath } = useI18next();
+  const { languages, language, originalPath, t } = useI18next();
+  const languageSwitchLabel = t('navigation.language-aria');
 
   return (
-    <div className={className}>
-      {languages.map((languageOfLink) => {
-        return (
-          <StyledLanguageLink
-            key={languageOfLink}
-            to={
-              language !== languageOfLink && translation
-                ? translation
-                : originalPath
-            }
-            language={languageOfLink}
-            selected={language === languageOfLink}
-          >
-            {languageOfLink}
-          </StyledLanguageLink>
-        );
-      })}
-    </div>
+    <nav aria-label={languageSwitchLabel} className={className}>
+      <ul>
```

Comment:
The nav element doesn't need the list. You could have swapped the div with the nav instead 🙏

Reply:
I wasn't sure about the list. Will remove it.